### ISSUE TITLE
OpenStackFloatingIP in_use method doesn't necessarily return the correct result

### ISSUE THREAD

**Issue by MartinPaulo** [gvlproject/cloudbridge#42]
When run against the nectar cloud the test_crud_network_service method fails as follows.

```bash
======================================================================
FAIL: test_crud_network_service (test.test_network_service.CloudNetworkServiceTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mpaulo/PycharmProjects/cloudbridge/test/helpers.py", line 72, in wrapper
    func(self, *args, **kwargs)
  File "/Users/mpaulo/PycharmProjects/cloudbridge/test/test_network_service.py", line 85, in test_crud_network_service
    "Newly created floating IP address should not be in use.")
AssertionError: Newly created floating IP address should not be in use.
```
When digging into the code it is found that the OpenStackFloatingIP in_use method tests the floating IP status, and if it is found to be 'ACTIVE' returns True. However, on our nectar cloud a newly created floating IP is set to 'ACTIVE' by default - even though it is not in use. A quick trawl through the neutron code seems to show that the status of a newly created floating IP depends on the underlying hardware. Hence ACTIVE is not a good indicator as to if the IP is in use or not.

**Comment by MartinPaulo** [comment]
I've been trying to work out what indicates if a neutron floating ip is in use or not. The guys here say "port_id is set". Looking at the shade code  [shade/shade/_normalize.py](https://github.com/openstack-infra/shade/blob/fcf76a9ca981ff4731e3c1f7b58729a04ed13f37/shade/_normalize.py) the `_normalize_floating_ip` method has the following line `attached = bool(port_id)`, so it looks like the guys are correct...

**Comment by afgane** [comment]
Fixed in #43.
